tests/integration/test_charm.py

```diff
@@ -12,19 +12,19 @@
 
 APP_NAME = "cos-config"
 PROM_NAME = "prometheus"
-SPRING_NAME = "spring-music"
+ZINC_NAME = "zinc"
 
 
 @pytest.mark.abort_on_fail
 async def test_dependencies(ops_test):
     await asyncio.gather(
-        ops_test.model.deploy("ch:prometheus-k8s", application_name=PROM_NAME, channel="beta"),
-        ops_test.model.deploy("ch:spring-music", application_name=SPRING_NAME, channel="edge"),
-        ops_test.model.deploy("ch:spring-music", application_name="spring-music2", channel="edge"),
-    )
-    await ops_test.model.wait_for_idle(
-        status="active", apps=[PROM_NAME, SPRING_NAME, "spring-music2"]
+        ops_test.model.deploy(
+            "ch:prometheus-k8s", application_name=PROM_NAME, channel="edge", trust=True
+        ),
+        ops_test.model.deploy("ch:zinc-k8s", application_name=ZINC_NAME, channel="edge"),
+        ops_test.model.deploy("ch:zinc-k8s", application_name="zinc2", channel="edge"),
     )
+    await ops_test.model.wait_for_idle(status="active", apps=[PROM_NAME, ZINC_NAME, "zinc2"])
```

Comment:
No need to provide `apps` arg if you're waiting for _all_ the apps.

Reply:
Done.